include/logger/loggerConfig.h

```diff
@@ -344,11 +344,13 @@ typedef struct _CANMapping {
 
     /* the conversion filter to apply to quickly convert units without changing mapping */
     uint8_t conversion_filter_id;
+
+    /* sub ID index. If defined (>=0) then this uses the first byte to mach on a sub address */
+    int8_t sub_id;
```

Comment:
So is this some standard or is this some device's special way of doing things.  If the latter the better place would be in the driver that handles that device.

Reply:
Also any reason for this to be a signed value?  Probably should be unsigned no?

Reply:
It's a quasi standard, so it should be part of the core capability. 

I'm using a singed value so I can use < 0 to indicate the feature is disabled, to prevent storing another flag.  I don't expect more than 127 sub channel IDs, so it should work well for an 8 bit signed.